### PR TITLE
feat(core): `Notification` add sizes

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/core/components/alert/alert.component.ts [hunk 1]
```diff
@@ -62,11 +56,7 @@ export class TuiAlertComponent<O, I> implements OnInit {
             return;
         }
 
-        timer(
-            tuiIsNumber(this.autoClose)
-                ? this.autoClose
-                : this.options.defaultAutoCloseTime,
-        )
+        timer(tuiIsNumber(this.autoClose) ? this.autoClose : 3000)
```
Comment (nsbarsukov) [inline projects/core/components/alert/alert.component.ts:59]: Why not `this.options.autoCloseTime` ?

```suggestion
        timer(tuiIsNumber(this.autoClose) ? this.autoClose : this.options.autoCloseTime)
```

Comment (vladimirpotekhin) [inline projects/core/components/alert/alert.component.ts:59]: breaking change ?

Comment (waterplea) [reply projects/core/components/alert/alert.component.ts:59]: I highly doubt somebody used that. There is already a prop to set autoClose duration.

Comment (waterplea) [reply projects/core/components/alert/alert.component.ts:59]: There is already a prop to set autoClose duration. It is basically a second layer level of abstracting this away. It is unnecessary and it is defined in notification options, not in alert options which is wrong.

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**